### ISSUE TITLE
Parse Error : syntax error, unexpected ';', expecting T_STRING on line 7 

### ISSUE THREAD

**Issue by aminechraibi** [VemtoOrg/vemto2-issues#32]
**Error Message:**
Parse Error : syntax error, unexpected ';', expecting T_STRING on line 7
   5 |
   6 |
>  7 | use ;
     |    ^
   8 | use Illuminate\Database\Eloquent\Model;
   9 | use Illuminate\Database\Eloquent\Factories\HasFactory;
  10 |

**Stack:**
```
SyntaxError: Parse Error : syntax error, unexpected ';', expecting T_STRING on line 7
   5 |
   6 |
>  7 | use ;
     |    ^
   8 | use Illuminate\Database\Eloquent\Model;
   9 | use Illuminate\Database\Eloquent\Factories\HasFactory;
  10 |
    at mt.raiseError (file:///F:/installed/Vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:271:175)
    at mt.error (file:///F:/installed/Vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:271:830)
    at mt.expect (file:///F:/installed/Vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:272:414)
    at mt.read_namespace_name (file:///F:/installed/Vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:260:37618)
    at mt.read_use_declaration (file:///F:/installed/Vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:260:38451)
    at mt.read_use_statement (file:///F:/installed/Vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:260:38029)
    at mt.read_top_statement (file:///F:/installed/Vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:267:1527)
    at mt.read_top_statements (file:///F:/installed/Vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:267:907)
    at mt.read_namespace (file:///F:/installed/Vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:260:36852)
    at mt.read_start (file:///F:/installed/Vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:260:36534)
```

**How to reproduce:**
1.  Adding attributes to User Model
2.  Save changes
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.0.5
win32 10.0.19045
Locale: fr
App version: 2.0.5

**Comment by TiagoSilvaPereira** [comment]
@aminechraibi Hi, what attributes did you added to the User model? Could you please share them?

**Comment by aminechraibi** [comment]
I recall creating a new model named 'User,' which initially contained only the fields: Id, created_at, and updated_at. I planned to add more attributes, such as name, email, and image. While configuring the model, I played with model options and don't remember exactly what those were. However, an issue arose when I attempted to save these changes. In response, I deleted the model and then closed and reopened the application. Subsequently, the default fields associated with the Laravel User model appeared, and I added the desired attributes again, and this time the saves worked successfully.

**Comment by TiagoSilvaPereira** [comment]
@aminechraibi, thanks for the details. I have a question: generally, a Laravel project already comes with a user table and a User model. Could you let me know if you removed them?

**Comment by aminechraibi** [comment]
No, I haven't deleted the user table. I may have overlooked it because I entered the **Laravel** Schema and started creating the Model without realizing the **App** schema was already there.

**Comment by TiagoSilvaPereira** [comment]
Hi @aminechraibi could you please check if this problem is still happening on 2.0.10 version?

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.11
